README.md

```diff
@@ -1,41 +1,33 @@
-# chromecast-sample-receiver
-This is the repository that contains an open-source Ooyala Chromecast sample receiver. This application should not be used for testing or evaluating Ooyala capabilities.
+# Chromecast Receiver + OOyala Player v4
+This repository contains the implementation of Google cast sdk as a receiver for video playback through Chromecast devices. It has an adaptation of OOyala Player v4.
```

Comment:
OOyala -> Ooyala

Reply:
There are a couple more places below too

Reply:
the block was removed